### PR TITLE
Artik images update, get-started nav, misc fixes

### PR DESCRIPTION
**Pull request by caldav**

### CODE REVIEW

File: templates/includes/markdown/get_started_first_boot.md [hunk 1]
```diff
@@ -18,15 +18,13 @@ the same network, using the following command:
...
   * During setup,` console-conf` will download the SSH key registered with your Store account and configure it so you can log into the device via `ssh <Ubuntu SSO account name>@<device IP address>` without a password.
   * There is no default `ubuntu` user on these images, but you can run `sudo passwd <account name>` to set a password in case you need a local console login.
 
-### Install and develop snaps
+## Install and develop snaps
 
-Your board is ready to have snaps installed.
-
-[Get started with the snap command](http://snapcraft.io/docs/core/usage)
+Your board is ready to have snaps installed, [get started with the `snap` command](http://snapcraft.io/docs/core/usage).
```
Comment (nottrobin) [inline templates/includes/markdown/get_started_first_boot.md]: Should this have a colon (:) or an em-dash (—)? E.g.: "...have snaps installed — get started..."? I feel the comma isn't quite the right punctuation.

File: templates/includes/navigation.yaml [hunk 2]
```diff
@@ -16,6 +16,14 @@ root:
 
 core:
   get-started:
+    installation-medias:
```
Comment (nottrobin) [inline templates/includes/navigation.yaml:19]: Shouldn't these follow the order of what's mentioned on the `/core/get-started` page? Or are you planning to change that page to reflect this new order & new items?

File: templates/includes/markdown/get_started_first_boot.md (hunk 1 above)
Comment (caldav) [reply templates/includes/markdown/get_started_first_boot.md]: Using &mdash; sounds good to me

File: templates/includes/navigation.yaml (hunk 2 above)
Comment (caldav) [reply templates/includes/navigation.yaml:19]: Two of the pages in the nav are not on the front page (first and last nav item), and are only referenced in boards pages. Having them this way is a logical order and gives quick access to the installation-medias page, without cluttering the message on the front page, which is a kind of a "supported boards" page.

**Review by nottrobin — APPROVED** [review]
LGTM

**Pull request merged**